### PR TITLE
BTFS-663: updated PR per comments

### PR DESCRIPTION
**Pull request by steveyeom**

### CODE REVIEW

File: splitting.go [hunk 1]
```diff
@@ -118,4 +124,45 @@ func (ss *sizeSplitterv2) Reader() io.Reader {
...
+	}
+}
+
+// Reader returns the io.Reader associated to this Splitter.
+func (ms *MetaSplitter) Reader() io.Reader {
+	return ms.r
+}
+
+// ChunkSize returns the chunk size of this Splitter.
+func (rss *MetaSplitter) ChunkSize() uint64 {
```
Comment (taiyangc) [inline splitting.go]: `rss` -> `ms` ?

Comment (steveyeom) [reply splitting.go]: opps sorry typo

Comment (steveyeom) [reply splitting.go]: updated.

**Review by taiyangc — APPROVED** [review]
lgtm

**Pull request merged**